nameko/testing/pytest.py

```diff
@@ -23,14 +23,29 @@ def pytest_addoption(parser):
         help=("The logging-level for the test run."))
 
     parser.addoption(
-        "--amqp-uri", action="store", dest='AMQP_URI',
-        default='amqp://guest:guest@localhost:5672/:random:',
-        help=("The AMQP-URI to connect to rabbit with."))
+        "--rabbit-host", action="store", dest='RABBIT_HOST',
```

Comment:
uri's are still a convenient way to specify driver (protocol), host, port and credentials in a single string. if we no longer allow the vhost to be specified i guess we can ignore it in the uri?

(even if we change the default, do we not even want to let users override the vhost?)